tcl/pdtk_textwindow.tcl

```diff
@@ -23,13 +23,17 @@ proc pdtk_textwindow_open {name geometry title font} {
             [concat pdtk_textwindow_close $name 1]
         bind $name <<Modified>> "pdtk_textwindow_dodirty $name"
         text $name.text -relief raised -highlightthickness 0 -bd 2 \
-            -font [get_font_for_size $font] \
+            -font [get_font_for_size $font] -exportselection 1 -undo 1 \
             -yscrollcommand "$name.scroll set" -background white
         scrollbar $name.scroll -command "$name.text yview"
         pack $name.scroll -side right -fill y
         pack $name.text -side left -fill both -expand 1
         bind $name.text <$::modifier-Key-s> "pdtk_textwindow_send $name"
         bind $name.text <$::modifier-Key-w> "pdtk_textwindow_close $name 1"
+        bind $name.text <$::modifier-Key-a> "pdtk_textwindow_selectall $name; break"
```

Comment:
it seems that there are default bindings here (Linux tcltk8.6) of <key>Ctrl</key>-<key>A</key> and <key>Ctrl</key>-<key>E</key> which move the cursor to the beginning resp. end of the current line. while i quite like the default behaviour, i have to admit that i only discovered it while testing this PR :-)

Reply:
We use MOD+A already for select all, so I was following that precedent. It's also what I'm used to and I think it would be helpful to be able to do it *somehow*.

Reply:
Annoyingly, those also work on macOS, except they are not mapped to the Command key.